### ISSUE TITLE
xilinx-linux and axis-fifo forked development

### ISSUE THREAD

**Issue by pfine91** [jacobfeder/axisfifo#27]
I am developing an application using the AXI4 FIFO and started by using the axis-fifo driver included in xilinx-linux.  I have found that modifications made in this repository are not in the xilinx-linux, and likewise, patches made to the xilinix-linux driver are not in this repository.

Has anyone made the effort to merge these two development paths back together?

**Comment by jacobfeder** [comment]
No, unfortunately I haven't had the time to keep them synced. If you would like to do that I'd be happy to give you write permissions.

**Comment by pfine91** [comment]
Thanks Jacob. I will let you know when I have a something to contribute.
